Sources/LogInOut.php

```diff
@@ -23,7 +23,7 @@
  *  It caches the referring URL in $_SESSION['login_url'].
  *  It is accessed from ?action=login.
  *
- * Uses Login template and language file with the login sub-template.
+ * @uses Login template and language file with the login sub-template.
```

Comment:
Incorrect use of tag; please revert

Reply:
Damn how did that get there.

Reply:
Reverted should be better now.

Reply:
I  guess you were working on an old file and staged too many changes at once.

Reply:
Yeah more than likely I thought just pulled it down but could have been comparing from an older github release with the files on my computer.